.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Summary
```

Comment:
This file is a magic Github feature. If this is merged, when someone opens a pull request it will automatically fill in this text for them to update. That way everyone will see the checkboxes at the bottom and know they need to test the changes.